roles/confluent.ksql/defaults/main.yml

```diff
@@ -30,20 +30,6 @@ ksql_service_environment_overrides:
   KSQL_OPTS: "{{ ksql_final_java_args | java_arg_build_out }}"
   KSQL_LOG4J_OPTS: "{% if ksql_custom_log4j|bool %}-Dlog4j.configuration=file:{{ ksql.log4j_file }}{% endif %}"
   # Remove trailing slash if there is one
-  # TODO investigate the ksql_log_dir variable wrt the log4j template
-  # LOG_DIR: "{{ ksql_log_dir|regex_replace('\\/$', '') if ksql_log_dir|regex_replace('\\/$', '') != ksql_default_log_dir else '' }}"
-  LOG_DIR: "{% if ksql_custom_log4j|bool %}{{ksql.appender_log_path}}{% endif %}"
+  LOG_DIR: "{{ ksql_log_dir|regex_replace('\\/$', '') if ksql_log_dir|regex_replace('\\/$', '') != ksql_default_log_dir else '' }}"
   CONFLUENT_SECURITY_MASTER_KEY: "{% if ksql_secrets_protection_enabled|bool %}{{secrets_protection_masterkey}}{% endif %}"
   ROCKSDB_SHAREDLIB_DIR: "{{ksql_rocksdb_path}}"
-
-ksql:
-  appender_log_path: /var/log/confluent/ksql/
```

Comment:
this change does not seem backwards compatible. ppl use the ksql.appender_log_path var today
better to deprecate it

Reply:
nvrm the var was just moved into vars/defaults/main.yml